website/docs/d/catalog.html.markdown

```diff
@@ -43,6 +43,9 @@ The following arguments are supported:
 ## Attribute Reference
 
 * `description` - Catalog description.
+* `publish_enabled` - (*v3.6+*) Enable external publication. Default is `false`.
```

Comment:
Here we definitely need more explanation of publication to where. Not all users know this feature.
Same ask for the resource part below.